src/src/rx_main.cpp

```diff
@@ -161,6 +161,39 @@ SerialIO *serialIO = nullptr;
 #define SERIAL_PROTOCOL_RX Serial
 #define SERIAL1_PROTOCOL_RX Serial1
 
+#if defined(RADIO_SX127X)
+// #define BEACON_DEBUG        // disable for flight use
+
+#if (defined(Regulatory_Domain_AU_915) || defined(Regulatory_Domain_FCC_915)) 
+    #define BeaconFrequency     915750000
+#elif (defined(Regulatory_Domain_EU_868) || defined(Regulatory_Domain_IN_866))
+    #define BeaconFrequency     868750000
+#elif (defined(Regulatory_Domain_AU_433) || defined(Regulatory_Domain_EU_433) || \
+        defined(Regulatory_Domain_US_433) || defined(Regulatory_Domain_US_433_WIDE))
+    #define BeaconFrequency     433750000
+#elif (defined(Regulatory_Domain_JP_920))
+    #define BeaconFrequency     920750000
+#else 
+    #error "Unsupported Regulatory_Domain defined. Add beacon frequency for it here."
+#endif
+
+#define BeaconFrequency     868750000
```

Comment:
How about using the `freq_center` from the `FHSSconfig` structure, or maybe add another element to the end with the beacon frequency. That way these defines are not needed.